nevergrad/optimization/base.py

```diff
@@ -621,6 +622,12 @@ def minimize(
 
 
 # Adding a comparison-only functionality to an optimizer.
+# Can be used as follows:
+# SQPOptimizer = SQP(50)   # Creates a SQP optimizer in a continuous domain for dimension 50 without constraints.
+# addCompare(SQPOptimizer)
+#
+# Then, SQPOptimizer can be used with SQPOptimize.compare([c1,c2,c3], [bc1, bc2, bc3]) instead of using "tell":
+# it will asssume that candidates c1, c2, c3 are better than candidates bc1, bc2, bc3.
 def addCompare(optimizer: Optimizer) -> None:
```

Comment:
If you plan on using it further, please:
- rename it with a correct case.
- use a docstring instead of comments

I still really don't like it though, because it is adding a method dynamically, and because this method is defined inline so would probably not be picklable, which will cause issues in the experiments. I suggest we talk through it a bit more, maybe we can come up with something better eventually.